Open_Acidification_pH-stat_arduino/Open_Acidification_pH-stat_arduino.ino

```diff
@@ -154,7 +135,6 @@ const int TEMP_CORR_ADDRESS = 12;
 const int KP_ADDRESS = 20;
 const int KI_ADDRESS = 28;
 const int KD_ADDRESS = 36;
-const int MAC_ADDRESS = 44;
```

Comment:
I think I would leave this since the MAC_ADDRESS is still being stored at this address. Think of this as part of the documentation.